src/App.js

```diff
@@ -1,15 +1,37 @@
 import React from 'react';
-import './App.css';
+import { useSelector } from 'react-redux';
 import { Routes, Route } from 'react-router-dom';
+import SplashScreen from './components/SplashScreen/SplashScreen';
 import Register from './components/Register/Register';
 import Login from './components/Login/Login';
+import Doctors from './components/Doctors/Doctors';
+import BookAppointment from './components/Appointments/BookAppointment';
+import MyAppointments from './components/Appointments/MyAppointments';
+import Navbar from './components/layouts/Navbar';
+import AddDoctor from './components/Doctors/AddDoctor';
+import DeleteDoctor from './components/Doctors/DeleteDoctor';
+import DoctorDetails from './components/Doctors/DoctorDetails';
 
 function App() {
```

Comment:
- [ ] For some reason, the images are not loading:

![image](https://user-images.githubusercontent.com/57421823/206867204-59067741-e2ea-4d1b-85f7-7283efffde6c.png)

Kindly correct this bug so your app shows as beautiful. 💪